### ISSUE TITLE
想问一下那个验证部分的pred=torch.argmax(torch.exp(pred),dim=1)的目的是不是把维度转换为和label一样大

### ISSUE THREAD

**Issue by porDame0** [MKaxie/MySegregation#3]

**Comment by MKaxie** [comment]
不是转化维度，是为了取每个像素点的预测结果
